pkg/profilecreator/profilecreator.go

```diff
@@ -172,13 +172,9 @@ func GetMCPList(mustGatherDirPath string) ([]*machineconfigv1.MachineConfigPool,
 	for _, mcp := range mcpFiles {
 		mcpName := strings.TrimSuffix(mcp.Name(), filepath.Ext(mcp.Name()))
 
-		// must-gather does not return the master nodes
-		if mcpName == "master" {
-			continue
-		}
-
 		mcp, err := GetMCP(mustGatherDirPath, mcpName)
-		if err != nil {
+		// master pool relevant only when pods can be scheduled on masters, e.g. SNO
+		if mcpName != "master" && err != nil {
```

Comment:
this may change soon enough in `control-plane`

Reply:
We only say "is not an error if the MCP hasn't been picked up by the must-gather".
What can we do better? How can we fail ?

Reply:
Not sure, I guess we can only track k8s/ocp and update the literal string.

Reply:
:(